add-on/panels/css/panel.css

```diff
@@ -96,6 +96,67 @@ body {
   border-radius: 3px;
 }
 
+/* Rename closing room panel */
+.rename-newRoom > img {
+  margin-left: 16px;
+  position: absolute;
+  width: 32px;
+}
+html[dir="rtl"] .rename-newRoom > img {
+  margin-right: 16px;
+}
+
+.rename-container {
+  margin: 16px;
+  /* margin is img-width + 16px separation */
+  padding-left: 48px;
+}
+html[dir="rtl"] .rename-container {
+  padding-right: 48px;
+}
+
+.rename-container > .rename-header {
+  color: #000;
+  font-size: 1.2em;
+  padding-top: 5px;
+}
+
+.rename-container > .rename-prompt {
+  margin-bottom: 8px;
+}
+
+.rename-container > .input-group {
+  background: #fff;
+  border: 1px solid #d1d1d1;
+  border-radius: 2px;
+  margin-left: -2px;
+  width: 100%;
+  padding: 2px 4px;
+}
+html[dir="rtl"] .rename-container > .input-group {
+  margin-right: -2px;
```

Comment:
Doesn't look like it'll matter much, but probably good practice to include `margin-left: 0;` here.